### PR TITLE
New Hardware section

### PR DESCRIPTION
**Pull request by cassidyjames**
Since we're starting to see hardware distributors ship elementary OS, we want to publicize our guidelines on the branding page.

### CODE REVIEW

**Comment by cassidyjames** [comment]
Available at https://beta.elementary.io/branding-hardware/brand#hardware

File: brand.php [hunk 1]
```diff
@@ -100,6 +100,19 @@
         <img src="<?=$sitewide['root'];?>images/brand/community-black.png" alt="elementary Community Logo in black"/>
         <p>This helps establish the product as part of the overall elementary community while reducing confusion that can arise from using the main logomark.</p>
     </section>
+    
+    <section id="hardware">
+        <h2>Hardware Distributors</h2>
+        <p>When a user sees or experiences elementary OS, we want to be sure they are seeing it as we intended. We also want to ensure they will have a consistent experience no matter how they access it, be it by downloading it from our website or using it pre-installed on a hardware product.</p>
+        <p>A modified version of elementary OS may be re-distributed pre-installed on a hardware product so long as it meets one or more of the following conditions:</p>
+        <ol>
+            <li>It is completely rebranded, containing none of the elementary brand marks,</li>
```
Comment (danrabbit) [inline brand.php]: Probably should add a line stating that they need to follow the terms of the licensing of the including software, specifically the GPL

File: brand.php [hunk 2]
```diff
@@ -100,6 +100,19 @@
         <img src="<?=$sitewide['root'];?>images/brand/community-black.png" alt="elementary Community Logo in black"/>
         <p>This helps establish the product as part of the overall elementary community while reducing confusion that can arise from using the main logomark.</p>
     </section>
+    
+    <section id="hardware">
+        <h2>Hardware Distributors</h2>
+        <p>When a user sees or experiences elementary OS, we want to be sure they are seeing it as we intended. We also want to ensure they will have a consistent experience no matter how they access it, be it by downloading it from our website or using it pre-installed on a hardware product.</p>
+        <p>A modified version of elementary OS may be re-distributed pre-installed on a hardware product so long as it meets one or more of the following conditions:</p>
```
Comment (danrabbit) [inline brand.php]: Maybe we should rephrase this a bit and put the focus on our brand. They can redistribute the software as long as they follow the terms of the software's licensing. But they can only redistribute our brand marks under the terms of their licensing, which is as long as the OS remains unmodified.

**Comment by danrabbit** [comment]
Something I just thought of is that we should make it easy to comply with removing trademarks if that's the route they choose. We should call out what exactly this entails and maybe even how to do it. As far as I remember, the only modifications that would be needed would be:

* LSB Release
* The boot/shutdown screen
* The `distributor-logo` icon

**Comment by cassidyjames** [comment]
Working on this directly with @danrabbit in a Google Doc. Will update the PR once we've settled on copy.

**Comment by cassidyjames** [comment]
Go go gadget @danrabbit

File: brand.php [hunk 3]
```diff
@@ -100,6 +100,23 @@
         <img src="<?=$sitewide['root'];?>images/brand/community-black.png" alt="elementary Community Logo in black"/>
         <p>This helps establish the product as part of the overall elementary community while reducing confusion that can arise from using the main logomark.</p>
     </section>
+    
+    <section id="hardware">
+        <h2>Hardware Distributors</h2>
+        <p>We want to ensure that as long as our software carries the elementary branding, the experience will be consistent whether it was downloaded it from our website or pre-installed on a hardware product.</p>
```
Comment (danrabbit) [inline brand.php]: it was downloaded it :p

**Comment by cassidyjames** [comment]
@danrabbit ^

**Pull request merged**